enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/CleanupCleanState.tsx

```diff
@@ -0,0 +1,15 @@
+import { t } from "ttag";
+
+import { CollectionEmptyIcon } from "metabase/collections/components/CollectionEmptyState/CollectionEmptyState";
+import { Flex, Text } from "metabase/ui";
+
+export const CleanupCleanState = ({ duration }: { duration: string }) => {
+  return (
+    <Flex align="center" direction="column" pt="10rem" pb="11rem">
+      <CollectionEmptyIcon />
+      <Text fw="bold" size="1.25rem" mt="4rem">
+        {t`All items have been used in the past ${duration}`}
```

Comment:
Since the translator might not be able to guess what the placeholder is about, think about adding some context with: `c("{0} is a duration").t(...`

Reply:
Something to bear in mind is that these context strings might one day be parsed by an AI translator - who will very likely benefit from having them, if they're clear.